.gitignore

```diff
@@ -1,2 +1,5 @@
 /build
 /node_modules
+
+*.iml
+npm-debug.log
```

Comment:
sunt fisiere care contin info despre proiectul din intellij idea, sunt specifice fiecarei instalari de IDE
eu in general incerc sa omit fisierele specifice sistemului de la includerea in surse

in ignore e bine sa fie orice fisier care nu tine de surse, chiar si astia de la github au sabloane de fisiere gitignore